apertium-eng.eng.rlx

```diff
@@ -329,6 +344,33 @@ SELECT V IF (-1 (prn pers)) ;
 SELECT SUB:1 Be IF (1 Ing) (0/* Have) ;
 SELECT Ger IF (-1/* Be) (1 To) ;
 
+SELECT Adv IF (0 Pr) (-1 Vblex) ;
+#Example (on):
+#He talked on.
+
+SELECT Cnjadv IF (0 Pr) (1 Det) (2 N) ;
+#Example (for):
+#Please get fish for the cat.
+
+SELECT Adv IF (0 N) (1 Supadj) ;
+#Example (smallest)
+#Canada is the second largest country in the world.
+
+SELECT Cnjadv IF (-1 Cm) (0 Pr) (1 Det) (2 N) ;
+#Example (for):
+#The cat didn't know, for the coconut didn't see.
```

Comment:
This will make the wrong decision for sentences like "I didn't bring a bag, but he did, for the coconut."  Also, relying on punctuation is probably not the best idea.

Reply:
I've removed this rule.